When making a new config with a server.json file that exists and has `{}` as its contents, mdmctl fails

### What version of `micromdm` are you using? 
Built from repo

### What `micromdm` command did you run? 
mdmctl

### What did you expect to see?
config set success (no response)

### What did you see instead?
panic: assignment to entry in nil map

goroutine 1 [running]:
main.saveServerConfig(0xc0000e1df8, 0x7ffc5d66653d, 0x8, 0x1a, 0x0)
	/home/peter/Work/micromdm/cmd/mdmctl/config.go:261 +0xc3
main.setCmd(0xc00001e0c0, 0x6, 0x6, 0x3, 0x0)
	/home/peter/Work/micromdm/cmd/mdmctl/config.go:164 +0x2e6
main.(*configCommand).Run(0xc00000e048, 0xc00001e0b0, 0x7, 0x7, 0xc0000e1e01, 0xc00000e048)
	/home/peter/Work/micromdm/cmd/mdmctl/config.go:62 +0xf5
main.(*configCommand).Run-fm(0xc00001e0b0, 0x7, 0x7, 0x6, 0xffffffffffffffff)
	/home/peter/Work/micromdm/cmd/mdmctl/mdmdctl.go:28 +0x48
main.main()
	/home/peter/Work/micromdm/cmd/mdmctl/mdmdctl.go:49 +0x185

--- a/cmd/mdmctl/config.go
+++ b/cmd/mdmctl/config.go
@@ -288,6 +288,9 @@ func loadClientConfig() (*ClientConfig, error) {
 	if err != nil {
 		return nil, fmt.Errorf("failed to unmarshal %s : %s", path, err)
 	}
+	if cfg.Servers == nil {
+		cfg.Servers = map[string]ServerConfig{}
+	}
 	return &cfg, nil
 }
 

